src/doc/partials/type.hbs

```diff
@@ -1,77 +1,81 @@
-            </p>
-        </div>
-    </div>
-
-    <div class="row">     
-        <div class="col-sm-24">
-         <h1 style="font-family: 'Selawik Light';">Selawik</h1>
-         </div>
-         <div class="col-sm-6">
-            <h1 class="win-type-headline" style="font-family: 'Selawik Light';">Headline</h1>
-            <h1 style="font-family: 'Selawik Light';">Header</h1>
-            <h2 style="font-family: 'Selawik Light';">Subheader</h2>
-            <h3 style="font-family: 'Selawik Semilight';">Title</h3>
-            <h4 style="font-family: 'Selawik';">Subtitle</h4>
-            <h5 style="font-family: 'Selawik Semibold';">Base</h5>
-            <p style="font-family: 'Selawik';">Body</p>
-            <p class="win-type-caption" style="font-family: 'Selawik';">Caption</p>
-            <p class="win-type-caption-alt" style="font-family: 'Selawik';">CaptionAlt</p>
-            <p class="win-type-subcaption" style="font-family: 'Selawik';">Subcaption</p>
-        </div>
-        <div class="col-sm-12">
-            <p style="font-family: 'Selawik';">
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. 
-            </p>
-            <p style="font-family: 'Selawik';">
-                Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?
-            </p>
-        </div>
-    </div>
 
-    <div class="row">
-        <div class="col-sm-12">
-            <h4>CSS classes</h4>
+        <div class="row">
+            <div class="col-sm-12">
+                <h4>CSS classes</h4>
 
             {{#markdown}}
 ```xml
-<h1 class="win-type-headline">...</h1>
+<h1 class="win-type-headline">Headline</h1>
 <div class="win-type-headline">...</div>
 
 <h1>Header</h1>
```

Comment:
win-type-headline goes a step larger than the default h1, but all the h1, .h1, h2, .h2 classes should work just fine.